src/catchem_config.F90

```diff
@@ -84,7 +84,7 @@ module catchem_config
     !integer :: num_plume_data  
   
   
-    real(kind=kind_chem) :: wetdep_ls_alpha(chem_tune_tracers)=-999.
+    real(rk) :: wetdep_ls_alpha(chem_tune_tracers)=-999.
```

Comment:
we can remove this.  wetdep isn't used in the dust schemes themselves.  separate routine for this